Doppler.Sap/Mappers/Billing/BillingForArMapper.cs

```diff
@@ -36,18 +36,21 @@ public bool CanMapSapSystem(string sapSystem)
 
         public SapSaleOrderModel MapDopplerBillingRequestToSapSaleOrder(BillingRequest billingRequest)
         {
+            var invoiceDate = billingRequest.InvoiceDate ?? _dateTimeProvider.GetDateByTimezoneId(_dateTimeProvider.UtcNow, _timezoneConfig.InvoicesTimeZone);
```

Comment:
what do you think add in this line the command `.ToString("yyyy-MM-dd")` ?

Reply:
Done